packages/graphback-runtime/src/createRuntimeServices.ts

```diff
@@ -0,0 +1,49 @@
+import { GraphQLObjectType, GraphQLSchema } from "graphql"
+import { PubSubEngine } from 'graphql-subscriptions';
+import { GraphbackDataProvider } from './data/GraphbackDataProvider';
+import { CRUDService } from './service/CRUDService';
+import { GraphbackModelConfig } from "./GraphbackModelConfig";
+
+export type RuntimeServicesOptions<Provider extends GraphbackDataProvider, Service extends CRUDService> = {
+  schema: GraphQLSchema,
+  models: GraphbackModelConfig[]
+  db: any
+  pubSub: PubSubEngine
+  provider: new (...args: any[]) => Provider,
+  service: new (...args: any[]) => Service;
```

Comment:
If we remove service and simply pass pubsub we will be sorted. Removing service though will require specific resolver wrappers as functions for pub sub and security later. Any customisation from users could be done by wrapping code into their apps. Is that making sense? would you like to use that kind of API yourself